app/main/views/communications.py

```diff
@@ -0,0 +1,84 @@
+from flask import render_template, redirect, url_for, current_app, \
+    request, flash
+from flask_login import login_required
+
+from .. import main
+from ..auth import role_required
+from ..helpers.communications import file_is_pdf, file_is_zip
+from ... import data_api_client
+from . import get_template_data
+
+from dmutils import s3
+
+
+def _get_path(framework_slug, path):
+    return '{}/communications/{}'.format(framework_slug, path)
+
+
+def _get_itt_pack_path(framework_slug):
+    return '{0}/communications/{0}-itt-pack.zip'.format(framework_slug)
+
+
+@main.route('/communications/<framework_slug>', methods=['GET'])
+@login_required
+@role_required('admin')
+def manage_communications(framework_slug):
+    communications_bucket = s3.S3(current_app.config['DM_COMMUNICATIONS_BUCKET'])
+    framework = data_api_client.get_framework(framework_slug)['frameworks']
+
+    itt_pack = next(iter(communications_bucket.list(_get_itt_pack_path(framework_slug))), None)
+    clarification = next(iter(communications_bucket.list(_get_path(framework_slug, 'updates/clarifications'))), None)
+    communication = next(iter(communications_bucket.list(_get_path(framework_slug, 'updates/communications'))), None)
+
+    return render_template(
+        'manage_communications.html',
+        itt_pack=itt_pack,
+        clarification=clarification,
+        communication=communication,
+        framework=framework,
+        **get_template_data()
+    )
+
+
+@main.route('/communications/<framework_slug>', methods=['POST'])
+@login_required
+@role_required('admin')
+def upload_communication(framework_slug):
+    communications_bucket = s3.S3(current_app.config['DM_COMMUNICATIONS_BUCKET'])
+    framework = data_api_client.get_framework(framework_slug)['frameworks']
+    errors = {}
+
+    if 'communication' in request.files and request.files['communication'].filename is not u'':
```

Comment:
This could be `if request.files.get('communication')` as the `FileStorage` object evaluates to `False` if there is no file.